src/main/minio.js

```diff
@@ -494,6 +494,12 @@ export class Client {
     if (!isValidBucketName(bucketName)) {
       throw new errors.InvalidBucketNameError('Invalid bucket name: ' + bucketName)
     }
+
+    if (isFunction(region)) {
+      cb = region
+      region = ''
+    }
```

Comment:
If `region` is a function, then it is set as a string, `region = ''`. So, there is no need to check if it is a string again.

`}` **=>** `} else if (!isString(region)) {`

Reply:
The current pattern is more readable @ebozduman i.e for argument validation we always use `if` statement